modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/dto/Account.java

```diff
@@ -62,26 +64,23 @@
     /** Reset password token. */
     private String resetPwdTok;
 
-    /** Registered. */
```

Comment:
Why this was removed? We have this field in MongoDB with date where user registered.

Reply:
This is not used in UI.